### ISSUE TITLE
Author is displayed incorrectly

### ISSUE THREAD

**Issue by Loughty** [fast4x/RiMusic#462]
### Steps to reproduce the bug

1. Search for the song "Say yes to heaven" and play it
2. See that the author is "Lana del Rey", add it to a playlist
3. Go to the playlist, and play that same song from there

### Expected behavior

Should say the name of the author

### Actual behavior

Shows "3:30" as author, which is the length of the song

### Screenshots/Screen recordings

![Screenshot_20240121-042904](https://github.com/fast4x/RiMusic/assets/52808915/432f94d8-ed3e-42cf-ad78-e5253b54cbb6)
![Screenshot_20240121-042918](https://github.com/fast4x/RiMusic/assets/52808915/a6f05233-ed51-43c5-904c-e5e9e3639862)

### RiMusic version

v0.6.18

### Android version

Android 12 Go

### Additional information

I remember this happened on the old ViMusic with the same song when it was new, but just with the update that orders the songs I remembered it because it was put first in my playlist

I cleared cache just in case

I tried different playlists, the same

**Comment by fast4x** [comment]
Thanks for reporting this issue

**Comment by fast4x** [comment]
Bug not reproduced, song are normally displayed and normally played 
![Screenshot_20240121_154920](https://github.com/fast4x/RiMusic/assets/1484476/826b4ea7-f0c6-41f2-b224-7c4cca77db8c)

**Comment by faustaahmad** [comment]
@Loughty perhaps try to send link to the playlist?
Maybe the bug occur when there are plenty of songs inside a playlist

**Comment by Loughty** [comment]
Sorry for the late reply!

> @Loughty perhaps try to send link to the playlist? Maybe the bug occur when there are plenty of songs inside a playlist

It's a local list, there's no option to share it as far as I know, but I recorded a video for them to see.

![screen-20240131-082801-2](https://github.com/fast4x/RiMusic/assets/52808915/dfb3a19c-ff9e-4648-9232-754cd99ac457)
(Sorry for the quality, GitHub doesn't want to play the video in MP4 for some reason, so I converted it to GIF)

Could it have something to do with the fact that I am using a ViMusic backup (I also created a new list, as seen in the video)? Any suggestions on how I could fix it or give you more information?
